### DISCUSSION TITLE
The signing became so slow and I have to request the minting again to make it work

### DISCUSSION THREAD

**Discussion by mohammed-io** [WalletConnect/walletconnect-monorepo#4108] [Developer Support]
Hello everyone,

Recently we are facing a lot of random behavior when it comes to minting (including the issues with iOS17). 

Currently, the minting is failing from the first time, however, it works and shows the modal if I go back to the app and request the minting again—which is a bad UX.

This is the repro video
https://github.com/WalletConnect/walletconnect-monorepo/assets/18808136/925ffdbe-1555-429c-9bbf-e67af82c2126

And the repo:
https://github.com/mohammed-io/wallet-connect-no-redirect-repro

**Comment by glitch-txs** [comment]
Hi could you test it here? https://lab.web3modal.com/

Which wallet are you using?

For iOS17 redirect from wallet to dapp is not supported anymore, wallets need to update their UI to tell the user to go back manually
